%%%%%%%%% PROJECT DESCRIPTION  -- 15 pages (including Prior NSF Support)

\required{Project Description}
\begin{center}
%\emph{Maximum of 6 pages}
\end{center}
%The Project Description (including Results from Prior NSF Support, which is limited to five pages) may not exceed 15 pages. Visual materials, including charts, graphs, maps, photographs and other pictorial presentations are included in the 15-page limitation. PIs be cautioned that the project description must be self-contained and that URLs that provide information related to the proposal should not be used. 

%All proposals to NSF are reviewed utilizing the two merit review criteria, intellectual merit and broader impacts. 

% The Project Description should provide a clear statement of the work to be undertaken and must include: objectives for the period of the proposed  work and expected significance; relation to longer-term goals of the PI's project; and relation to the present state of knowledge in the field, to work in progress by the PI under other support and to work in progress elsewhere.



%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
%INTRO
% a brief and informative introduction or background section
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%

\section*{A. Introduction}

A critical goal of evolutionary biology today is to understand how organisms are able to adapt to new or changing conditions. This is especially relevant in the face of global climate change and other increasingly common anthropogenic changes to the environment (cite climate change papers). Understanding the process and the outcomes of adaptation can also further inform crucial goals such as improving crop yields.

% talk about demographic history broadly and the difficulties it causes
The demographic history of populations or species plays a major role in the strength of selection on traits and organisms (cite), and thus has the ability to leave lasting changes or signatures in the genome (cite). A large body of work aims to infer past demographic histories, such as bottlenecks, repeated founder effects during expansion, or rapid population growth, of populations based on signatures in the genome (cite). However, inferring demographic history is a difficult task, as there are many possible histories that may have occurred, and they have the potential to leave similar, or even identical, signals in some species (cite\kjg{e.g. stationary pop growth looks same as range expansion unless you look at the right parameters}). Knowing such demographic history, however, is key, as it has a direct relationship with population size. For example, population bottlenecks greatly decrease the effective population size ($N_e$) which can increase the effect of random genetic drift. The effect size, or strength of selection ($s$), for a given locus, can also impact the efficiency of selection across the genome (selection effective when $N_es>1$). Thus, as demography changes, so does $N_es$ and therefore so does the occurrence of different beneficial, deleterious, or neutral loci across the genome.

% talk about genetic architecture broadly and how important it is to understand for adaptation etc 
The complexity of demographic history is increased manyfold when considering the various architectures that species' genomes comprise. Many ecological traits important for local adaptation have a complex, quantitative genetic basis, and much heterogeneity has been found among these traits both within and among species (cite Orr 2001, Slate 2005). Whether this genetic architecture consists of many loci of small effect or few of large effect can therefore play a role in the impact of demography and selection, as well as mutation, on the genome. How the genome may be restructured as a result of various demographic histories and those effects on selection is not well studied. The genetic architecture underlying traits can affect how easily or quickly local adaptation may occur, or how long and difficult that process may be (cite Yeaman?). Such knowledge can therefore greatly contribute to improve breeding and conservation efforts as well as for predicting responses to environmental changes.

% now into the specifics of this research proposal
My proposed research aims to investigate these important factors (demographic history, selection, and genetic architecture) and their effects and interactions in terms of shaping the genome. I will address these questions using the maize/teosinte system. Maize (\emph{Zea mays}, ssp. \emph{mays}) is the domesticated species commonly known as corn, an annual plant. It is known to have been domesticated from its ancestor, the wild teosinte, approximately 9,000 years ago in southwestern Mexico. In particular, subspecies \emph{Zea mays parviglumis} is the direct progenitor of maize. While two other subspecies of wild teosinte, \emph{mexicana} and \emph{huehuetenangensis}, exist and are found at higher elevations on the Mexican Central Plateau and in western Guatemala, respectively (cite Hufford 2012). Maize and teosinte are extensively studied and serve as ideal models for this research. Demographic history of the species is well documented: supported by both archaeological and other human records of its use and domestication in the Americas (cite). Extensive genetic and genomic data is also available, allowing the known demographic history of the species to be assessed in terms of genomic signatures of domestication (cite?), as well as specifically parameterized in terms of important values such as effective population sizes (cite Beissinger in prep). This provides a unique opportunity to compare these closely related subspecies which have undergone various different demographic processes contributing to their present-day genetic make-up.

In this research project I propose three objectives which make use of the extensive knowledge and data available in the maize\//teosinte system in order to answer the question of \textbf{how the genetic architecture of traits changes as a result of demography and selection}. Objective one uses existing genomic data in teosinte to characterize the genetic architecture of phenotypically important traits. Using the results of this study for parameterizing key genomic characteristics, objective two then simulates the demographic history of teosinte's domestication into modern-day maize. Objective three then characterizes the genetic architecture of the same traits using genomic data from modern maize and compares to theoretical expectations of the genetic architecture of these traits based on the results produced from objective two's simulations. This research will further our understanding of demography and selection's impact on the genome and its architecture, as well as our ability to detect and predict changes in genetic architecture after certain demographic events.



%it is thought, and shown in some human pops, that demog. history such as expansion leads to an increase in delet alleles, and of larger effects - b/c of continued inferred expansions and bottlenecks. 
%is there any evidence of this in maize?
	



%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
%OBJECTIVES
% a statement of research objectives, methods, and significance
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%

\section*{B. Research Objectives, Methods \& Significance}
\subsection*{Objective I: Investigate the genetic architecture of important traits in teosinte}

This first objective aims to estimate the distribution of fitness effects (DFE) in teosinte (\emph{genus spp}).


\subsection*{Objective II: Simulate the demographic history of maize domestication from teosinte}
Maize was domesticated in...




Small population sizes can lead to purging (recessives become homozygous and removed in smaller pops), but also increase in deleterious alleles (surfing phenomenon due to random genetic drift).
mutations that affect a trait related to fitness will then be impacted by demography x selection interaction.
in annual plants, nearly ALL traits related to fitness
we thus predict that genetic architecture (number and size of mutations) should be impacted by demographic change.
this is not understood well in any system. controversial in humans (lohmueller vs. pritchard etc.). in plants, summaries are on gross overall $N_e$ (small vs. big) and ignore recent demographic change.





\subsection*{Objective III: Estimate and compare the genetic architecture underlying traits in maize post-domestication from simulations and genomic data} 
 rationale and significance










\begin{comment}

Some such quantitative phenotypes include yield, plant height, and flowering time, which are of critical importance to agriculture.  

Quantitative phenotypes such as yield, plant height, and flowering time are of critical importance to agriculture.  
Deleterious alleles likely play a large role in many of these phenotypes: crop plants have undergone dramatic demographic shifts, usually involving a domestication bottleneck followed by expansion as cultivation spread, and some authors even argue that selection on domestication traits has inadvertently increased the frequency of alleles deleterious for other phenotypes (cite gunther2010). 
Consistent with this hypothesis, my lab has recently shown that genes associated with a number of quantitative traits in maize are enriched for deleterious alleles  compared to randomly chosen genes (cite mezmouk2014).
However, while we know that demography impacts the frequency of individual deleterious variants, we have a poor understanding of the interaction of demography and selection on phenotypic variation. 
In particular, we know little about how these two forces interact to determine the genetic architecture -- the number of genes and their effect -- of a trait. 
Such information is crucial for understanding variation in phenotype, designing breeding strategies, utilizing diversity from wild relatives, or even engineering new traits using biotechnology. 

	

Maize was domesticated $\approx$9000 years ago in SW Mexico which bottlenecked populations to a small size, but was subsequently followed by a large population expansion more recently. This demographic event can be detected genetically, as shown by Beissinger in prep (github repo), where populations were bottlenecked to 5\% of their previous size followed by spread across central and southern America and into highland and lowland environments. Populations of maize have since recovered to much larger effective population sizes than even before the domestication bottleneck (humongous growth to at least 300K but maybe as much as 1E9, citation).

Being a crop species that is incredibly useful to humans (lots of citations and examples of usage), there is also a wealth of knowledge on quantitative traits. 
will need to explain rare alleles pops and experiment some (PGRP15 grant)
%using genomic data from both maize and teosinte across their species' ranges, we can compare
%	classes of functional diversity, using e.g. GERP scores, across pops of different demographic history
%	look at the distribution of these as well as in beneficial traits if possible
%can compare these results to simulations parameterized to match known demog. of teosinte/maize

This system thus serves as a great resource to compare the effects of this history on the genome in maize versus its extant, wild ancestor teosinte, particularly to investigate how the genetic architecture of these important traits may have evolved differently as a result of this combination of demography and selection.


\subsection{Obj 1 - Estimate the Distribution of Fitness Effects (DFE)}

	using teosinte genomes \kjg{note to self, update the data description}\\
		- from polymorphism/divergence data. use HapMap2 or current teosinte genomes (I think I'd vote for latter)
		\url{http://goo.gl/CLmsmX} and \url{http://www.genetics.org/content/177/4/2251.short})
		can either use estimated demographic model or use noncoding sites to normalize SFS
		
	Methods: estimate the DFE using Eyre-Walker's DoFE \url{http://www.lifesci.susx.ac.uk/home/Adam_Eyre-Walker/Website/Software.html}
	
\kjg{this part prob not in proposal:}
	can validate by comparing to
		GERP distribution
		partitioning variance components, e.g. \url{http://www.ncbi.nlm.nih.gov/pubmed/25439723}
		GREML software
				
%	\kjg{this gets at magnitude of effect, but also want to get at how many loci may be contributing to any given trait?}
%    \jri{nope. we just need DFE. see obj 2}

	useful because the distribution of mutation effect sizes is not generally known, and is especially difficult for small effect mutations.
	objective 1 will inform perhaps what the DFE may look like in any organism with a history similar to teosinte? 
	and just in general add to the body of literature on genetic architecture, mutation effects

	
\subsection{Obj 2 - Simulate scenarios of different traits.}
	
	from objective 1 we now know the DFE of teosinte
	we already know the demographic history of maize since its split from teosinte (citations)
	
	use the DFE results to parameterize a model that will simulate the evolution of maize and its genetic architecture through time during and since its domestication
	we can simulate maize that expanded into S America separately since it has a different demography and then compare any differences the regions may show in the end
		simulate traits w/ varying correlation with fitness
		new mutation effects on fitness determined by DFE, effect on trait by correlation between trait and fitness
	
	Methods: fwdpy (python version of fwdpop, cite \url{http://www.genetics.org/content/198/1/157.abstract}), allows simultaneous generation of complex demography, natural selection, and quantitative phenotypes, including deleterious mutations and their effect on phenotype
	
	evaluate:
		how many loci contribute to important traits?
		how strong are these effects?
		how do details of demography impact outcome?
		test against theory e.g \url{http://arxiv.org/abs/1312.3028}
	
	standing questions:
		does the DFE significantly change in a meaningful way or a certain direction?
			mean value the same but narrower or wider distribution?
			skewed more one way or the other?
		might expect this to be a different answer for maize in S America vs Central since S America has had a second bottleneck, so more founder effects and more chance for drift
	
	(could also do some broader simulated examples just to stand alone and see if other various outcomes may occur - just don't plan on comparing these to any real data)\jri{yes, once we can show we can recapitulate real data, i think this is useful}

\subsection{Obj 3 - compare simulation results to modern maize genomes, known to have undergone the same demographies simulated in objective 2}%
	compare to GWAS for maize/teo. do we recapitulate observations?
		several traits are genotyped/phenotyped in maize/teosinte
		If the estimated demographic model and DFE are reasonable, the genetic architecture of simulated phenotypes should closely mimic that of real data.
	are there differences between central and southern American pops? \jri{we have no GWAS data for S.Amer. pops, but do have genomes and GERP. we could get freq. etc. of del. mutations from sims and compare to GERP}
	
	Methods: same DoFE approach in C American maize pops for direct comparison
		in S American pops, can do comparison on subset, e.g. GERP, which we would have from Obj 1 if we compare to other approaches for the sort of validation of the DFE
			(definitely worth doing if b/c of 2nd bottleneck more deleterious stuff rose in frequency and was then eliminated)

% imp. point from Jeff's service.tex file
	If the real data differ from simulation, we can explore the sensitivity of genetic architecture to changes to the demography or DFE; understanding this sensitivity will then lead to improved estimation of these important parameters.

\end{comment}	
	
	
	
	
	
	
	
	
	
	
	
	
	
	
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
%TRAINING OBJECTIVES
% training objectives and plan for achieving them (these may include scientific as well as other career preparation activities)
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
\section*{C. Training Objectives}

Hmmm, seems somewhat related to D, need to differentiate and write more about training here, may lead to changes in D.

%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
%CAREER DEVELOPMENT
% an explanation of how the fellowship activities will enhance your career development and future research directions as well as describing how this research differs from your dissertation research, thus providing you an opportunity to broaden your scientific horizon
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
\section*{D. Career Development \& Future Research}

My career goal is to become an independent, academic researcher who is able to push the boundaries of population genetics and evolutionary biology. As I have in my dissertation research, I aim to study population genetic and genomic processes both empirically from real work data and theoretically through simulations. I believe one of the strongest ways to advance our knowledge is through such comparisons of situations where every parameter, current and historical, is known, as well as the evolutionary outcome (simulations) to natural situations where processes or effects that are poorly understood or still unknown to us can diverge from theoretical expectations and provide the basis for further study and investigation into these processes.

The skills I will develop during this fellowship will benefit my career and put me on the cutting edge for analyses of the newest genomic data and the most recent computational approaches for biological simulations. First, I will gain many skills related to genomic data analysis throughout the course of this research. I have minimal exposure and experience directly working with such data from my dissertation, thus making htis a vital opportunity. Genomic technology and data are growing at an incredibly fast pace, and working directly with such data will teach me the most up to date, accurate, and efficient approaches. I will also improve my skills of computational biology through the proposed simulations and be able to learn a new and useful programming language used widely in evolutionary biology, Python. Drs. Ross\--Ibarra and Thornton are both at the forefront of a popular \kjg{think of better word?} movement for open science, making all stages of the research process transparent to any interested parties, and providing products such as data and code immediately and publicly. This is a work ethic I strongly agree with and hope to contribute to as an independent researcher. Our work together will better equip me with the tools and experience that make open science easy, efficient, and profitable for all. Interacting with Drs. Ross\--Ibarra and Thornton, as well as other researchers at UC Davis, on a regular basis will be both intellectually stimulating and rewarding experiences that will help me accomplish my career goals.


%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
% HOST INSTITUTION
% a justification of the choice of sponsoring scientist(s) and host institution(s)
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
\section*{E. Sponsoring Scientists and Host Institution}

The University of California Davis (UCD) is the ideal place to conduct the proposed research. UCD has a world-renowned program in evolutionary biology and faculty in population genetics who are at the top of the field. Jeff Ross\--Ibarra is an expert on teosinte, maize, its domestication, and the associated population genetics and genomics of the system. \kjg{Jeff, feel free to make that sound better} 
Kevin Thornton is an accomplished quantitative geneticist and computational biologist who will also contribute greatly to this research. \kjg{Likewise Kevin, feel free to modify}
They will both serve as effective and capable mentors for my post-doctoral research. In particular, Jeff has been studying the maize\//teosinte system for \emph{XX}\kjg{don't forget to fill in} years with a great network of collaborators providing vast resources of data. His work has contributed largely to our knowledge of this system, and more generally on domestication and adaptation as evolutionary processes. Kevin is also the developer and maintainer of fwdpy, the python package proposed for completing the simulations in Objective 2. He will thus serve as a great resource in terms of knowing the exact capabilities of the simulation method and any assumptions of its model that must be taken into account.
Furthermore, the Department of Ecology and Evolution, the Department of Plant Biology, and the Department of Plant Sciences at UCD have many exceptional faculty doing research relevant to my interests, providing many research groups to interact with on a daily basis for potential collaborations or feedback on this research. For example, I look forward to interacting with scientists interested in population genetics, such as Graham Coop, and in adaptation, such as Johanna Schmitt. %Additionally, experts in plant regulatory evolution, such as Neelima Sinha, Siobhan Brady, and Daniel Runcie will serve as great people to interact with. % need to look up people in the dept, these names were just taken from Emily J's application
UCD has the necessary computing resources for our proposed work, and as described, vast sources of knowledge and experience on the topics I plan to investigate, ensuring the success of this work. I am excited to join and contribute to UCD's active and vibrant scientific community.


%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
%Milestones
% a timetable with yearly goals with benchmarks for major anticipated outcomes
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
\section*{F. Milestones \& Timeline}
\begin{tabular}{ll}
Year 1 \hspace{0.5in} & accomplish this\\
Year 2                     & Finish stuff  \\
Year 3		& I don't think there is a year 3?\\
\end{tabular}
\kjg{want to do the DFE of maize and teosinte at the same time? no reason we can't right, and then do the simulations. Though if it takes time, teosinte should be prioritized so the sims can be parameterized and started while the maize DFE is estimated}

%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
%BROADER IMPACTS
% a separate section within the narrative that describes in detail the broader impacts of the proposed activities.
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
\section*{G. Broader Impacts}

The proposed research will have wide-ranging impacts for both the public and the scientific community.


genetic architecture underlying traits affects how easy/hard and quick/slow local adaptation can occur. important for breeding, conservation, predicting response to climate change, etc.
useful info for crops/domesticated species (and also things like disease in humans?)

\begin{comment}
Quantitative phenotypes such as yield, plant height, and flowering time are of critical importance to agriculture.  
Deleterious alleles likely play a large role in many of these phenotypes: crop plants have undergone dramatic demographic shifts, usually involving a domestication bottleneck followed by expansion as cultivation spread, and some authors even argue that selection on domestication traits has inadvertently increased the frequency of alleles deleterious for other phenotypes (cite gunther2010). 
Consistent with this hypothesis, my lab has recently shown that genes associated with a number of quantitative traits in maize are enriched for deleterious alleles  compared to randomly chosen genes (cite mezmouk2014).
However, while we know that demography impacts the frequency of individual deleterious variants, we have a poor understanding of the interaction of demography and selection on phenotypic variation. 
In particular, we know little about how these two forces interact to determine the genetic architecture -- the number of genes and their effect -- of a trait. 
Such information is crucial for understanding variation in phenotype, designing breeding strategies, utilizing diversity from wild relatives, or even engineering new traits using biotechnology. 
\jri{see ideas and text in ``service\_award.tex'' that I uploaded too.}
\end{comment}




