%%%%%%%%% PROJECT DESCRIPTION  -- 15 pages (including Prior NSF Support)

\required{Project Description}
%\begin{center}
%\emph{Maximum of 6 pages}
%\end{center}
%The Project Description (including Results from Prior NSF Support, which is limited to five pages) may not exceed 15 pages. Visual materials, including charts, graphs, maps, photographs and other pictorial presentations are included in the 15-page limitation. PIs be cautioned that the project description must be self-contained and that URLs that provide information related to the proposal should not be used. 

%All proposals to NSF are reviewed utilizing the two merit review criteria, intellectual merit and broader impacts. 

% The Project Description should provide a clear statement of the work to be undertaken and must include: objectives for the period of the proposed  work and expected significance; relation to longer-term goals of the PI's project; and relation to the present state of knowledge in the field, to work in progress by the PI under other support and to work in progress elsewhere.

%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
%INTRO
% a brief and informative introduction or background section
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%

\vspace{-0.5cm}

\section*{A. Introduction}	% might be a little long currently, should not repeat the summary page

A critical goal of evolutionary biology today is to understand organisms' abilities to adapt to new or changing conditions. This is especially relevant in the face of global climate change and other increasingly common anthropogenic changes to the environment \citep{Easterling:2000ja}. Most ecologically or economically important phenotypic traits have a complex, quantitative genetic basis, and considerable heterogeneity exists within and among species in this genetic architecture---the number of causal loci, their effect sizes, and allele frequencies \citep{orr:2001, slate:2005}. Yet we know very little about how the genetic architecture shapes, and is shaped by, the process of adaptation. In order to understand the maintenance of variation or adaptive potential of phenotypes of interest, we must understand how their genetic architecture changes as a result of evolutionary processes such as selection and demography, and how differences in genetic architecture may limit adaptive change. Such knowledge advances our understanding of the process of adaptation and can further benefit crucial goals such as improving crop yields under changing climates. 
% \jri{see changes to objective \ref{surfing} addition talking about limiting?}

%Our current knowledge of the genetic basis of adaptation is surprisingly limited. \kjg{this sentence repeats the last paragraph}
Nearly all knowledge of adaptation comes from population genetic theory focusing on single loci or quantitative genetics theory that evaluates variances and ignores the underlying loci. Yet theoretical analysis of simple models of quantitative traits have shown that neither of these approaches can successfully predict the impact of evolutionary processes on the genetic diversity of loci underlying quantitative traits \citep{Chevin:2008, LeCorre:2012co}. 
While quantitative trait locus (QTL) mapping approaches and genome-wide association studies (GWAS) have successfully identified a number of causative loci, these methods suffer from statistical issues limiting their utility for studying the overall genetic architecture of adaptive traits \citep{spencer2009designing,Gibson:2012,Thornton:2013} 
%adaptation  mapping approaches cannot achieve the resolution required to separate causal loci. While genome-wide-association studies (GWAS) improve resolution but suffer from multiple testing difficulties that lead to failure to detect loci of small effect. Furthermore, it is not well understood which or if there may be a general regime for the genetic architecture of important phenotypic traits. Many common alleles of small effect (the infinitesimal model) is difficult to detect, as just described, but another possibility of many very rare alleles of large effect is also difficult because detection would require sufficient sampling to accumulate enough of these rare variants in the sample . 
%\kjg{commented out what was there, tried to get at what I thought was meant here, feel free to edit again}
% Many studies in plants suffer from relatively small population sizes\kjg{is this true? if it's a matter of rare things and small pops can anything fix this? not just an inability of the methods in general?}, and are thus unable to asses the potential importance of rare mutations of large effect \citep[cf][]{Thornton:2013}.  \krt{Thornton et al. don't get into sample size and power.  A better citation would be }
%impacts of both  also been criticized for its
%Both genome-wide association (GWAS) and QTL approaches are limited by inability to detect small effect alleles. QTL studies are not able to achieve the resolution necessary for distinguishing the number and effect size of contributing loci, while GWAS studies fail to explain all phenotypic variance, questionably missing either rare large effect alleles or common small effect alleles \citep{, }.
Alleles missed in most QTL or GWAS studies may nonetheless be of central importance to adaption \citep{Rockman:2011ej, Mackay:2009}, and novel approaches are thus needed to understand how evolutionary processes affect the architecture of quantitative traits.  

The genetic architecture of a trait is determined by a number of factors, including population size, mutation, demographic history, and the history of both purifying and positive selection. Population bottlenecks, for example, can lead to the loss of variation or maintenance of deleterious variation \citep[e.g.][]{Renaut:2015hi, Gunther:2010}. Strong selection, such as that imposed during crop domestication, can fix large-effect loci \citep{Brown:2011}. Both theoretical and empirical studies in humans have shown that the genetic architecture of human traits is shaped by the frequency and effect size of mutations \citep{Thornton:2013} as well as the interaction of demographic and selective histories \citep{Fu:2014jt, Gravel:2011iq, Henn:2015dp}.
%\jri{i like this paragraph but not sure if it stays here or moves} - kjg I vote stays, last paragraph ends discussing the need to better understand evoln & gen arch, and this para. details that more

% \jri{stoppign here but i think we redo this paragraph to be about quant gen of adaptaton rather than why qtl/gwas don't work. thoughts?}
%for studying the adaptation of quantitative traits  genetic architecture of traits are limiting.  

This proposal aims to develop a simulation framework that allows explicit modeling of the evolution of the genetic architecture of a quantitative trait. This model will then be used to investigate how different factors shape the architecture of quantitative traits in maize and its wild relative teosinte. %the evolution of model of genetic architecture, which if parameterized well can elucidate the complex genetics underlying changes in phenotypes. \kjg{don't quite have the words the way I want here, but trying to get the point across that we have real data before and after but can't get at the details of genetic architecture. then add simulations, and if we can get them to match the real data before and after, presumably we can use our omniscience in these simulations to tell us about what's happening with genetic architecture}
Thanks to its economic importance and history as a model genetic organism, extensive genomic \citep{Wright:2005,Hufford:2012dy} and phenotypic \citep{Wallace:2014,Weber:2009} data exist for both maize and its wild ancestor, teosinte, making maize an ideal system to in which to study processes affecting genetic architecture.  
%Previous work, for example, has estimated the demographic effects of domestication \citep{Wright:2005} and identified loci under selection \citep{Hufford:2012dy}. And a number of studies have used GWAS to study the architecture of phenotypes of interest in both maize \citep{Wallace:2014} and teosinte \citep{Weber:2009}, allowing us to compare our simulated and theoretical results with empirical estimates for both taxa.
\textbf{I propose four objectives which make use of the extensive knowledge and data available in the maize\//teosinte system in order to develop an explicit genetic model of the evolution of a quantitative trait.} %\jri{or bold this one?} yeah, I like this one bolded better - kjg
% \textbf{how demography and selection change the genetic architecture of quantitative traits}. 
Objective \ref{modeling} builds a simulation model fit to empirical estimates of the genetic architecture of phenotypically important traits in the wild ancestor of maize. Objective \ref{domestication} then uses simulations of maize domestication and comparisons to empirical data to understand how demography and selection combined affect %I could be wrong but I think affect not effect is the right word here?
the genetic architecture of phenotypes in maize.  Objective \ref{validation} empirically tests  predictions made in Objectives \ref{modeling} and \ref{domestication} by comparing the effects of maize and teosinte alleles in a common genetic background in two mapping populations. Finally, Objective \ref{surfing} uses our simulation model to predict changing genetic architecture in maize landraces as they spread across the Americas post-domestication. This research will further our understanding of the impact demography and selection have on phenotypic traits and their underlying genetics, improving our ability to predict the effects of selection and changing environments as well as to exploit genetic diversity in crops for continued breeding. This study and this system have tremendous opportunity to expand our knowledge of the genetic architecture of complex traits and to build a strong foundation for future studies in this direction.\kjg{I'll work on this, I feel like these sentences always sound so corny.} \jri{agreed. could almost end with the penultimate sentence? but yeah these sentences always seem cheesy}

%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
%OBJECTIVES
% a statement of research objectives, methods, and significance
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%

\section*{B. Research Objectives, Methods \& Significance}\vspace{-1ex}

\renewcommand\thesubsection{\Roman {subsection}.}

\subsection{Model the genetic architecture of phenotypes in teosinte}\vspace{-1ex}
\label{modeling}
In Objective \ref{modeling} I will build a model to simulate the genetic architecture of phenotypes in the wild ancestor of maize, teosinte (\emph{Zea mays} ssp. \emph{parviglumis}). Such a model will then enable us to study the impacts of demography and selection on the genetic basis of phenotypic traits during domestication (Objective \ref{domestication}) and range expansion (Objective \ref{surfing}).

In order to model the architecture of quantitative traits, we first need to understand the effects of mutation. The distribution of fitness effects (DFE) describes the consequences of mutations in terms of their impacts on an organism's fitness. Taking advantage of published \citep{Chia:2012} and ongoing whole-genome sequencing data in teosinte (see Data Management Plan), I will estimate the DFE in teosinte with the software DoFE \citep{Keightley:2007hq, Stoletzki:2011}.%\kjg{removed some detail here, could put back}%, using estimates of the number of nonsynonymous and synonymous substitutions in genes.
The resulting estimates of the fitness effects of new mutations can then be parameterized in terms of their effect on fitness-related quantitative traits \citep{Keightley:1988, eyre-walker:2010}, including yield, plant height, and flowering time.

%DELETED:Mutations can broadly be classified as beneficial, deleterious, or neutral, but in actuality, mutation effect sizes span a continuum of strongly deleterious to strongly beneficial, with any value in between. %good stuff, but probably not needed

%DELETED:Estimate of the DFE and effect size of new mutations will then be used 
%s well as new seqI will apply this method to sequence data from regions of the genome known to underly important phenotypic traits. Some such quantitative phenotypes include yield, plant height, and flowering time, which are of critical importance to agriculture (cite). From this distribution of fitness effects and fitness measures from these traits, we can then translate these values into effect sizes per allele.\kjg{eek, not sure I'm describing this well} I will use genotypic and phenotypic data on teosinte which is available for 5,000 individuals at 16 phenotypic traits (height, kernel traits...\kjg{do you think I should list them all here?}) (cite data source). These individuals are the progeny of 70 teosinte individuals sequenced to 25X coverage, providing an ideal resource for this analysis.

This estimated DFE, combined with prior information on mutation \citep{Clark:2005} and recombination rates \citep{Rodgers-Melnick:2014}, will then be used as input in simulation models of quantitative trait evolution in teosinte. Additional parameters necessary for our model (including dominance and the correlation between a trait and fitness) will be estimated using an Approximate Bayesian computation (ABC) approach \citep{Beaumont:2002ue}. I will draw simulation parameters from a prior distribution and then compare summary statistics generated by simulation to observed data. By keeping the parameter draws that generate data most similar to the observed data, I can estimate the parameters' posterior distributions. I will take advantage of the library fwdpy (a Python implementation of fwdpp \citealt{Thornton:2014kn}, available at \url{https://github.com/molpopgen/fwdpy}) to write simulation code that explicitly models a quantitative trait evolving under a model of stabilizing selection. From each simulation, I will then perform genome-wide-association analysis in order to compare model results with observed data from published \citep{Weber:2009} and on-going (see Data Management Plan) analysis of 16 phenotypic traits in a natural population of teosinte. ABC will allow robust estimation of the necessary parameters from a set of several million such simulations.
%\kjg{do you think we need to detail the summary stats used to pick the best fit of the model to the data? or that's basically what the GWAS is doing, right?}
%\jri{do we need more detail on fwdpy? size of region etc? you had this and i commented out.}\kjg{happy to leave out for space-saving, NSF doesn't need this level of detail, right?}

\subsubsection*{Significance}\vspace{-1ex}
Obtaining the DFE for teosinte provides a distribution of effect sizes for loci across the genome, information that is not widely available in many systems. Actual estimates of this distribution contribute to future studies aiming to realistically simulate any distribution of mutation effects by further describing the variability of this distribution across species, and these results add to important bodies of work such as understanding the genetic basis of complex diseases, among others \citep{EyreWalker:2007dl}. 

%\jri{need final paragraph here, see latex for comments} 
%NEED new paragraph on 1) why having the DFE (which is just distribution of s, not genetic architecture itself) is cool. see eyre-walker NatRevGen 2007. 
%	2) the fact that a model like this currently doesn't exist! no plug-and-play software, very limited theory. so just finishing objective 1 will be awesome!

%DELETED:the genomic architecture desired: distribution of effect sizes across the desired number of loci, including deleterious mutations and their effects on the phenotype. Values of $V_G$, $V_A$, and $V_D$ can be run to equilibrium, and then the populations sampled in order to test for successful replication of a wild teosinte population.

%DELETED:Characterizing the DFE of teosinte and translating this into a distribution of effect sizes uncovers whether the genetic architecture of important quantitative phenotypic traits in teosinte are underlain by many loci of small effect, few loci of large effect, or any combination therein. 
%DELETED:Estimating the DFE in teosinte will further inform a larger body of work aimed at understanding how common different genetic architectures are for traits important in adaptation. The DFE is difficult to estimate and not broadly understood in evolutionary biology for any given organism or in terms of how much it may vary across organisms. This difficulty arises particularly for cases where there are many loci of small effects: such small effect sizes are difficult to detect on an individual basis. 
		
\subsection{Model quantitative genetics of maize domestication}\vspace{-1ex}
\label{domestication}
The model developed in Objective \ref{modeling} will enable simulation of quantitative traits under stabilizing selection. Here I will build upon this model to incorporate both demographic change and directional selection to investigate the impact of domestication on the genetic architecture of phenotypes of interest. Both demographic change associated with a domestication bottleneck and directional selection are expected to change the architecture of quantitative traits, but current theory includes only relatively simple models, and a detailed understanding of how these two forces interact is lacking. 

First, I will extend the simulation framework developed in Objective \ref{modeling} to include estimates of population size change during the domestication bottleneck and subsequent expansion \citep{Wright:2005}. Using the parameters estimated for teosinte, I will simulate the same quantitative traits in a population undergoing a domestication bottleneck. This first analysis provides a null expectation of the genetic architecture of traits in the absence of directional selection. I will perform GWAS on the simulated data and compare these results to GWAS in both published \citep{Wallace:2014} and our own on-going analyses of maize data (see Data Management Plan). Differences between simulated and observed data are informative for the history of selection on phenotypes. I can again use an ABC approach in fwdpy to estimate the strength of selection on domestication phenotypes and re-simulate with more accurate selection parameters to assess how traits experiencing varying strengths of selection and with varying heritability have changed over the course of domestication.

%\jri{tempted to add GCTA-GREML from Yang 2011 PubMed ID: 21552263 for variance prediction here. argue GWAS is imperfect way of estimating these things, but I can look at distribution of how variance is partitioned. i suspect this might be too much and proposal would benefit more from extra clarity than extra methods.}

Traits believed to evolve neutrally during domestication may only be impacted by demographic forces of the domestication bottleneck, while other traits are thought to have experienced directional selection simultaneously. More heritable traits or traits under stronger directional selection may change more dramatically than those evolving neutrally while subject to the bottleneck. I can thus test if allelic effect sizes have shifted  their distribution mean or overall shape to more or fewer loci of large effect \citep{Chevin:2008} depending on the strength of selection. Traits in maize such as kernel weight and kernel row number, among other ear-related traits (Figure 1), are expected to have been selected for crop improvement, while there are no strong reasons to suspect other traits such as total plant height were under selection. This suite of traits thus provides us with \emph{a priori} predictions on which traits should show differences from our simple demographic simulations, and in cases where these predictions are not matched may indicate genetic correlations with selected traits that could be further investigated by study of the G matrix in maize. \kjg{wonder if this might be a case where it's too much jargon for reviewers without defining the G matrix? could just be vague and say "...could be further investigated."}

%\kjg{More heritable traits or traits under stronger positive selection may change more dramatically or if interact w/ demog in weird ways, then could change in unexpected ways? - i.e. if demog sends in one direction but seln senda back in the other , could cancel out instead of build up}

%\jri{maybe add that differences for traits not thought to be under selection might be explained by genetic correlations with other traits? we could test this by estimating G matrix in maize.}

% results may be attributable to parameter values for selection for which we can then reestimate using ABC, and then resimulate the domestication.  Once closely matched, I can make a second comparison of how the genetic architecture changed over time. 
%The distribution of effect sizes established in objective one are used now to simulate the domestication of maize from teosinte. 
%Again using fwdpy, I will simulate the demographic and selective events that occurred during the domestication of maize. A first step to fit the distribution of effect sizes from objective one will use approximate Bayesian computation (ABC) within fwdpy to estimate the most likely distribution of parameters for $V_G$ and $V_A$.\kjg{did I describe this right?} Simulations will follow the genetic architecture of the aforementioned quantitative traits and their accurate genetic architectures through a strong bottleneck of 5\% of its ancestral $N_e$. Regions of the genome in sizes of 100kb can be simulated with assigned fitness values. Population sizes are easily controlled within fwdpy to mirror the ancestral $N_e$ of maize to its estimated size of $\sim$120,000 followed by the domestication bottleneck which was subsequently followed by a large, rapid growth in population size that expanded populations to at least three times as large as the ancestral $N_e$ (cite). 

\vspace{-0.4cm}

\begin{SCfigure}[][h]
%\jri{see commented tex code to place exactly here, but i like better the autoplacing}
%	kjg - I tend to prefer the figure not appear before being mentioned in the text, but don't mind if you want it back the other way
%\begin{SCfigure}
\centering
   \includegraphics[width=0.55\textwidth]{Figure2_Brownetal2011.png}
  \caption{Effect sizes from GWAS analyses in maize, grouped by trait category. Inset shows the largest effects. Ear traits, showing the largest effect sizes, were likely selected during maize domestication while the others were not. Figure from \citealt{Brown:2011}.}
  \label{fig:model}
\end{SCfigure}

\vspace{-0.4cm}	% was starting to run past 6 pages, can remove this if things are okay in the end

%Having reiterated the origin of modern-day maize in Mexico through simulations, I can now compare the resultant genetic architectures to real genetic architecture in maize data. Genotypic and phenotypic data is currently being generated for another 5,000 progeny individuals of maize, from one landrace of 55 parents (sequenced to 25X) that will be available by the start of this fellowship. Existing data is also available publicly for $\sim$1500 maize genomes through the HapMap 3 and 4 projects (cite) if any unforeseen causes slow the availability of this dataset. 

%\krt{If anyone knows the Simons paper, this section may be puzzling.  Demog doesn't affect mean VG nor the burden, which is the result of Simons.  But, what is not well-understood is how the distr. of effect sizes is changes.}  
% kjg - yeah I guess I was just trying to highlight that I don't think this is a settled debate and that our results could confirm one way or the other. e.g. the excoffier group still works assuming that demog will affect load, e.g http://biorxiv.org/content/early/2015/05/22/019711 but maybe I should believe that less since these other ones are actually published.
The results of this Objective will show the relative importance of demography and selection in determining maize's genetic architecture. Demographic bottlenecks are common during the geographic spread of populations and can have several effects on the genome including purging of deleterious alleles (recessive alleles become homozygous and are more efficiently removed), or alternatively may lead to an increase of some deleterious alleles through increased random genetic drift (allele surfing, \citealt{Klopfstein:2005bl}). The effects of these processes vary depending on the degree of population size reduction and the length of time over which populations are bottlenecked \citep[e.g.][]{Caplins:2008ke}, but also have the potential to interact with the strength of selection during this demographic process. Changes in the distribution of allele effect sizes are not well understood in any system and are controversial in humans \citep{Hancock:2011jb, Henn:2015ce, Henn:2015dp, Lohmueller:2014dn, Simons:2014fj}. Deleterious alleles likely play a large role in many adaptive phenotypes: crop plants have undergone dramatic demographic shifts, usually involving a domestication bottleneck followed by expansion as cultivation spread, and some authors even argue that selection on domestication traits has inadvertently increased the frequency of alleles deleterious for other phenotypes \citep{Gunther:2010}. Consistent with this, it has recently been shown that genes associated with a number of quantitative traits in maize are enriched for deleterious alleles compared to randomly chosen genes \citep{Mezmouk:2014jd}. Such information is crucial for understanding variation in phenotype, designing breeding strategies, utilizing diversity from wild relatives, or even engineering new traits using biotechnology. 

%\jri{this is great. not sure if it goes here or in project significance?}\kjg{agreed, maybe move to end of part B}\jri{could be shortened too} -- kjg decided to leave for now, I think I like emphasizing how each objective/project has a role to play in contributing something broadly

\subsection{Validate model predictions in synthetic mapping populations}\vspace{-1ex}
\label{validation}

%\jri{this feels short. do we need to add more? explain better?} 
% I think it's okay since it's pretty straightforward in terms of applying GWAS again just now on these samples + a few more traits
Models developed in Objective \ref{modeling} and \ref{domestication} lead to direct predictions of the genetic architecture of quantitative traits and how they should differ between maize and teosinte. To validate the predictions of these models, I will take advantage of two mapping populations containing mixtures of teosinte and maize alleles: a set of near isogenic lines of teosinte alleles introgressed into a maize background \citep{Liu:InPress}, and a synthetic population resulting from the cross of 11 teosinte and 26 maize parents (see Data Management Plan). Genotype and phenotype data are available for both populations, and if necessary I will add additional genotyping to augment marker density.
% \jri{we can exclude this bit, i added mostly to help justify why they should give you research money but reality is we won't need to do this} << I like having it, means I'll get more genomics/wet lab experience 
I will perform a GWAS on these samples for the traits already used to calibrate our models. Identification of GWAS hits in regions of the genome coming from maize and teosinte will allow us to compare the effect size distribution of causal variants in a common environment and genetic background. With information on the correlation between phenotype and fitness, our model and simulations should be able to predict the difference in genetic architecture for new traits. I will thus perform a similar analysis for traits not included in our calibration to test the predictive ability of our model. 

\subsection{Investigate the impact of demography and local adaptation of maize landraces across the Americas} \vspace{-1ex}
\label{surfing}

%\jri{do we want to include the SeeDs phenotype/genotype data?} - kjg i think yes, but was there enough there to do much?
After domestication, maize spread across Central and South America, adapting to a variety of environments across a wide elevational range. These populations have experienced further demographic and selective pressures in addition to the initial domestication bottleneck. For example, South American populations are inferred to have experienced a second severe bottleneck \citep{Takuno:2015eu}, and more generally populations expanding geographically are likely to have experienced serial founder effects that can change allele frequencies in unexpected ways due to allele surfing \citep{Klopfstein:2005bl}. The effect of range expansion on deleterious alleles remains controversial \citep{Henn:2015ce, Henn:2015dp, Simons:2014fj, Sudmant:2015}, and even less is known about its impact on quantitative traits. Changing selection pressures from both new environments and artificial selection by differing human populations likely also interact with these demographies. 

Following Objective \ref{domestication}, I will simulate quantitative traits in populations undergoing further selection and demographic change. This will require new simulations to incorporate range expansion or other demographies as well as multiple shifts in selection. Though current data likely does not provide the resolution to estimate these parameters for all maize landrace populations, qualitative predictions about changes in genetic architecture can be compared to GWAS results from publicly available genotype and phenotype data of nearly 5000 landraces from across the Americas \citep{Hearne2015}. This will allow assessment of how complex demography and changing selection regimes affect the genetic architecture of local adaptation to different conditions. Additionally, I can test the ability of various genetic architectures that may have resulted from these past events in terms of their adaptability for the future \citep{Yeaman:2015cc}. I can impose identical shifts in selection on each architecture to see which has the greater or faster response. Understanding the characteristics that lead to this improved ability to adapt allow more predictability of evolutionary processes and better methods of artificial selection or conservation efforts to ensure the persistence of crops and other species of importance for biodiversity into the future. \kjg{just added these last 3 sentences after thinking about your comment in the intro - would be cool to do, but don't necessarily have to write them in here right now, plus has probably been somewhat done already}\kjg{I need to update intro line 47 if this is kept}


% latex FTW :)

%DELETED Maize originated approximately 9,000 years ago in southern Mexico (cite Matsuoka et al 2002, others?) during a single domestication event of \emph{ssp. parviglumis}. Archaeological records also confirm this dating and single location (cite) as well as the subsequent spread and growth in population size of domestic landraces across the Americas into both lowland and highland environments [cite Wilkes, H. G. (1967) Teosinte: The Closest Relative of Maize (Harvard Univ., Cambridge, MA)]. South American landraces of maize also underwent a second bottleneck event during their expansion (cite). Using genetic data, the precise demographic parameters of this history have been estimated (cite Beissinger et al in prep). This provides information on the ancestral effective population size of maize ($N_a \approx$ 120,000), the size to which the population was bottlenecked during domestication (5\% of this $N_a$), the subsequent size to which populations rapidly expanded (3 times as large as $N_a$\kjg{but maybe as much as 1E9, citation}), and lastly on the genome-wide mutation rate ($3.8\times10^-8$ cite Clark et al 2005 MBE 22, 2304 -- 2312.)

%DELETE? These individuals will occupy populations that will be subjected to combinations of demographic and selective pressures including some of the following. To recapitulate the various landraces of maize, I will include cases of a second bottleneck after the domestication bottleneck, populations undergoing little or significant additional range expansion (Central American versus South American), stronger or weaker selection on flowering time and phenological traits (warmer lowland versus colder highland adapted populations), and cases with or without gene flow from sympatric\kjg{is that accurate, there is teosinte around them at least?} populations of teosinte.
%\jri{ambivalent about the teosinte idea. on one hand very cool and lots of additional questions -- adaptive introgression, maladaptive gene flow, etc. on the other hand, further complication and another layer to add to already complex grant. inclined to skip for space reasons.} -agreed-kjg

%DELETED: The objective of this project is to simulate the impact of multiple demographic events and selective pressures either individually or in combination. This will show how much variation is possible in terms of the effects on genetic architecture underlying traits for adaptation. The core set of simulations in this project will be designed to match the known landraces in maize, but will include more broadly a designed experiment to compare the presence or absence of particular evolutionary forces. Using genomic and phenotypic data for some landrace populations of maize (see Data below), I can compare a subset of these simulations to real data to assess how well these simulation results match real populations across the species range. The goal of this project is to inform our understanding of the relative importance or insignificance of the tested demographic cases and how sensitive genetic architecture is to these.

%\subsection*{Data}
%This project will make use of both published data as well as data generated or currently being generated as part of a related Plant Genome project (Biology of Rare Alleles IOS-1238014) of which sponsoring scientist Dr. Ross-Ibarra is a Co-PI. As part of this project, Dr. Ross-Ibarra's group is currently in the process of sequencing 70 teosinte and 55 maize genomes to high depth. These sequences should be complete by the end of 2015 and will be made publicly available early 2016 via the group website (\url{www.panzea.org}). The individuals used for genome sequencing are also the parents of two large mapping populations of $\sim$5000 progeny. Both populations have been genotyped and phenotyped for a number of traits including seed yield, flowering times, and plant height, and would be used for comparisons in Objectives \ref{modeling} and \ref{domestication}. Finally, the same project has developed, genotyped, and phenotyped a  synthetic population of 11 teosinte and 26 maize parents, resulting in a population that is roughly 12\% teosinte genes, 40\% B73 (the reference genome line), and 2\% from 25 other inbred maize lines. This latter population will be used in Objective \ref{validation} to compare model predictions to maize and teosinte regions segregating in a common population.  All of these data are currently available from collaborators.  Publication of results from analyses of these data must await publication of collaborators' GWAS analyses, but as these are currently underway and comparisons with empirical data would likely not begin until early 2017 at the soonest we do not foresee this being an issue. In the unlikely event of a problem in the generation of that data, existing data is also available publicly for teosinte \citep{Weber:2009} and maize \citep{Wallace:2014} as well as $\sim$1500 maize genomes through the HapMap 3 project \citep{Bukowski:2015} and 4,000 landraces each with 1 million SNPs and several phenotypes \citep{Hearne2015}.


% NB if you prefer \approx, let me know! I've just been changing them to \sim because in my head I read that as "approximately" whereas I read the former as "approximately equal to", but that may just be me

%_________________% DATA INFORMATION %_________________%

%%%RARE ALLELES PROJECT (probably the most useful data)
% 1 pop of teo w/ 70 parents (sequenced), 5000 progeny (16 phenotypes, GBS, imputation in progress)
% 1 pop of landrace maize w/ 55 parents (sequencing in progress), 5000 progeny (~25 (?) phenotypes, GBS, imputation in progress)
% 4 additional teo pops,  each with 10 parents (sequencing in progress) and 1200 progeny (GBS done, phenotyping in progress done in Jan 2017)
% 4 additional landrace maize pops,  each with 10 parents (sequencing in progress) and 1200 progeny (GBS done, phenotyping in progress done in Jan 2017)

%%OTHER DATA OF USE
% for teosinte architecture
% Weber 2009 teosinte population (if DNAs still available; may be too small sample size pop) 

%for architecture of traits in maize

% NAM!! 5000 RILs, GBS, 42 phenotypes    26 parents crossed to an additional line, 15 million snps


%(suboptimal) alternative DFE, diversity data
% ~1500 sequnced maize genomes (Hapmap 3 and 4) 

%for comparison of maize and teo architecture
% teosinte synthetic: ~2500 DH lines, ~12% teosinte, 40% B73, 2% of each other 25 NAM parents there is dna and tissue
% Briggs 2007 mapping population (600 BC2S3 to estimate ), maybe genotyped on ~1500 markers
% Sherry parviglumis NILs (~800??) there are seeds

% for looking at other landraces
% ~6 landrace genomes each from highland mexico, lowland mexico, highland S. america, lowland s. america, highland guatemala, US southwest
% SeeDs: 
	% 4,000 landraces each w/ 1M SNPs and several phenos (publicly available)
	% 25,000 landraces, each w/ pooled seq. of ~30 plants. + some phenos (not yet avaible, but we know the woman in charge of the program)
	
% other goodies
% 0.2cM-scale recombination map
% rho-map (starting Feb. 2016)
%___________________________________________________%	

	
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
%TRAINING OBJECTIVES
% training objectives and plan for achieving them (these may include scientific as well as other career preparation activities)
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
\section*{C. Training Objectives}

This fellowship will provide me with an ideal opportunity to learn the skills needed to enhance my ability to conduct cutting edge research in the fields of genomics and computational biology, both areas in which I expect to continue my future research and which are greatly expanding in evolutionary biology. I will gain many skills related to genomic data analysis through these projects, learning bioinformatics and analysis skills for large datasets. I have limited experience working with genomic data from my dissertation, thus making this a vital step in my career. Genomic technology and data are growing at an incredibly fast pace, and working directly with such data will teach me the most up to date, accurate, and efficient approaches. I will also improve my computational biology skills through the proposed simulations, learning a new and useful programming language, Python, that I can apply throughout this research and my future research in evolutionary biology. 


%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
%CAREER DEVELOPMENT
% an explanation of how the fellowship activities will enhance your career development and future research directions as well as describing how this research differs from your dissertation research, thus providing you an opportunity to broaden your scientific horizon
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
\section*{D. Career Development \& Future Research}

My career goal is to develop an innovative research program in evolutionary biology, studying population genetics and processes that impact genetic diversity. I believe such research is key for the future, not only for the field of evolutionary biology, but also in applied scenarios such as understanding prevalence of genetic diseases in humans, adaptation of species to climate change, or strategies to improve agricultural products for a growing world population. My dissertation research has approached some of these questions in a more theoretical and less applied mindset. The work I will conduct during this fellowship would have more direct potential for application in the field of maize agriculture. For me, this is necessary and vital experience for my career development as I decide between pursuing a more applied research program, potentially in industry or government research scientist positions, or in pursing a career as an academic researcher at a university.

The skills I will develop during this fellowship, as described in section C, will benefit my career and put me on the cutting edge for analyses of the newest genomic data and the most recent computational approaches for biological simulations. Interacting with Dr. Ross\--Ibarra, as well as other researchers at UC Davis, and with Dr. Kevin Thornton at UC Irvine, will be both intellectually stimulating and rewarding experiences that will help me accomplish my career goals. Drs. Ross\--Ibarra and Thornton are both at the forefront of a popular movement for open science, making all stages of the research process transparent to any interested parties, and providing products such as data and code immediately and publicly. This is a work ethic I strongly agree with and hope to contribute to as an independent researcher. Our work together will better equip me with the tools and experience that make open science efficient, and rewarding for all. I believe that this will equip me as a competitive, knowledgeable, and independent researcher able to conduct interesting and useful research throughout my future research program on topics of local adaptation, demographic history, population structure and genetic architecture of important traits. Furthermore, Dr. Ross-Ibarra has an excellent track record of helping his post-doctoral fellows secure promising positions for their future careers, including 3 assistant professorships at universities, 2 research scientist positions in the seed industry, one at an NGO, and one in the USDA. 

%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
% HOST INSTITUTION
% a justification of the choice of sponsoring scientist(s) and host institution(s)
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
\section*{E. Sponsoring Scientists and Host Institution}

The University of California Davis (UCD) is the ideal place to conduct the proposed research. UCD has a world-renowned program in evolutionary biology and faculty in population genetics who are at the top of the field. Jeff Ross\--Ibarra is an expert on teosinte, maize, its domestication, and the associated population genetics and genomics of the system. Kevin Thornton is an accomplished quantitative geneticist and computational biologist at UC Irvine, who will also contribute greatly to this research. They will both serve as effective and capable mentors for my post-doctoral research. In particular, Jeff has been studying the maize\//teosinte system for many years with a great network of collaborators providing vast resources of data. His work has contributed largely to our knowledge of this system, and more generally on domestication and adaptation as evolutionary processes. Kevin is also the developer and maintainer of fwdpy, the python package proposed for completing the simulations. He will thus serve as a great resource in terms of knowing the exact capabilities of the simulation method and any assumptions of its model that must be taken into account.
Furthermore, the Department of Ecology and Evolution, the Department of Plant Biology, and the Department of Plant Sciences at UCD have many exceptional faculty doing research relevant to my interests, providing many research groups to interact with on a daily basis for potential collaborations or feedback on this research. For example, I look forward to interacting with scientists interested in population genetics, such as Graham Coop, and in adaptation, such as Johanna Schmitt. %Additionally, experts in plant regulatory evolution, such as Neelima Sinha, Siobhan Brady, and Daniel Runcie will serve as great people to interact with. % need to look up people in the dept, these names were just taken from Emily J's application
UCD and UC Irvine both have extensive computing resources for our proposed work with access to dedicated processors and nodes across multiple clusters with large RAM capabilities (see Sponsoring Scientist Statement for details), and as described, vast sources of knowledge and experience on the topics I plan to investigate, ensuring the success of this work. I am excited to join and contribute to such an active and vibrant scientific community.

%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
%Milestones
% a timetable with yearly goals with benchmarks for major anticipated outcomes
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
\section*{F. Milestones \& Timeline}
\begin{tabular}{ll}
Year 1: & Estimate DFE in teosinte; design model \& perform ABC (Obj. I) \\
 & Manuscript on genetic architecture of important phenotypic traits in teosinte \\
Year 2: & Model domestication \& selection, validate with empirical data (Obj. II, III)\\
 & Manuscript on evolution of genetic architecture during domestication of maize \\
Year 3: & Model local adaptation, population expansion (Objective IV)  \\
 & Manuscript on impact of demography and selection across maize landraces \\
\end{tabular}


%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
%BROADER IMPACTS
% a separate section within the narrative that describes in detail the broader impacts of the proposed activities.
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
\section*{G. Broader Impacts}

The proposed research will have wide-ranging impacts for both the public and the scientific community. I will ensure that my results are available to the public at all stages of these projects by maintaining code and scripts online at my GitHub account, which will allow other researchers to access analysis methods or data cleaning tools as well as simulation details and parameters which can provide a building block from which further research can be conducted. I will present new findings at international conferences and submit publications to open-access pre-print servers. I will also be able to broadcast my work more widely to the public through a strong online presence I maintain on Twitter and \href{http://www.molecularecologist.com/}{The Molecular Ecologist}, a blog I have contributed to in the past. I will also contact local schools where I can tailor seminars to students of different age ranges and topics of interest from general science of where crops and domestic species that we all live with and survive on come from to more in-depth presentations of my research and evolutionary biology as a field for high school students interested in pursuing further study in biology or the sciences more broadly. The impacts of this research on corn as a crop species useful for both food and fuel resources is also of an innate broad impact, as understanding the genetics underlying adaptation will ensure a viable future for such crop species in the future. Lastly, this research will contribute greatly to my own career development, improving my knowledge on genomics and working in an economically important crop species. I will be able to learn these vital tools as well as to teach them to undergraduate students in the lab and into the future as the field of genomics continues to grow.








